src/main/java/com/hubspot/maven/plugins/dependency/management/DependencyManagementAnalyzer.java

```diff
@@ -24,9 +25,10 @@
   private boolean dependencyVersionDisallowedError = false;
   private List<String> errorMessages = new ArrayList<String>();
 
-  public DependencyManagementAnalyzer(MavenProject project, RequireManagement requireManagement, Log log) {
+  public DependencyManagementAnalyzer(MavenProject project, RequireManagement requireManagement, boolean fail, Log log) {
```

Comment:
I think that it would make sense to add getters to `DependencyManagementMojo` and pass that down rather than having a long list of parameters. @jhaber is this something that's kosher in the maven plugin world?

Reply:
I guess it also doesn't quite work for https://github.com/HubSpot/dependency-management-maven-plugin/pull/8/files#diff-ffc9e225a0091ac8fe5f4e2d54a3d732R23